cli/tests/integration_tests.rs

```diff
@@ -1796,13 +1796,6 @@ itest!(_033_import_map {
   output: "033_import_map.out",
 });
 
-itest!(import_map_no_unstable {
-  args:
-    "run --quiet --reload --importmap=importmaps/import_map.json importmaps/test.ts",
-  output: "import_map_no_unstable.out",
-  exit_code: 70,
-});
```

Comment:
Why remove this test?

Reply:
This is tested in `flags` unit tests now. I also added `.requires("unstable")` on `--import-map` which wasn't there before, I suspect that's why this was mistakenly tested in integration tests.

Reply:
Nice 👍  thanks

Reply:
Oh nevermind I never added a unit test for that flag dependency, but like every other unstable flag I think there's no need for an integration test as long as it's represented in `flags`.